assets/images/icons/airport.svg

```diff
@@ -1,95 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Wouldn't we want to keep these svg's? In case we change anything with the theme/colors, etc, or want to make small changes to stuff. I'd be inclined to just leave them in there, even if we don't really need them for much of anything.